xbmc/cores/VideoPlayer/VideoRenderers/OverlayRendererGUI.cpp

```diff
@@ -136,6 +136,10 @@ COverlayText::COverlayText(CDVDOverlayText * src)
 
   m_type = TYPE_GUITEXT;
 
+  // entire area of screen
```

Comment:
"screen" is the wrong word. target area can be video control in Kodi. the OS Window of Kodi might not cover the full monitor either.